src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -156,14 +156,14 @@
   <!-- TODO create fwlink for this HelpUrl -->
   <BoolProperty Name="Deterministic"
                 DisplayName="Deterministic"
-                Description="Indicates whether the compiler should produce identical assemblies for identical inputs."
+                Description="Compilation produces identical output for identical inputs."
```

Comment:
Wouldn't this be something like "Produce identical compilation output for identical inputs." to match the structure of the others?